vxyowsup/whatsapp.py

```diff
@@ -63,9 +64,9 @@ def handle_outbound_message(self, message):
             return self.publish_nack(message['message_id'], 'failed')
             return self.publish_ack(
                 message['message_id'], 'remote-message-id')
-        # assumes message['to_addr'] will be the phone number
+        # assumes message['to_addr'] will be the phone number + '@s.whatsapp.net'
         self.stack_client.send_to_stack(
-            message['content'], message['to_addr'] + '@s.whatsapp.net')
+            message['content'], message['to_addr'])
```

Comment:
I think it would be cool to keep the `to_addr` the MSISDN and remove the `@s.whatsapp.net` before sending incoming messages out.